spec/requests/pseuds_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe "Pseuds", type: :request do
+  subject { page }
+
+  describe "show" do
+    let(:user) { FactoryBot.create(:user, login: "Agrajag") }
+
+    before do
+      FactoryBot.create(:pseud, user: user, name: "Zaphod")
+      FactoryBot.create(:pseud, user: user, name: "Slartibartfast")
+      FactoryBot.create(:pseud, user: user, name: "Betelgeuse")
+    end
+
+    it "shows only the configured maximum number of pseuds" do
+      allow(ArchiveConfig).to receive(:ITEMS_PER_PAGE).and_return(3)
+
+      visit "/users/Agrajag/pseuds"
```

Comment:
Similar to the thread at https://github.com/otwcode/otwarchive/pull/3813#discussion_r414959949, should we start using request specs here?

Reply:
Is the question about request versus view specs versus other kind of RSpec integration test, or is it about writing cucumber features instead?

I can't remember why I didn't make them controller specs - I think I tried and couldn't surmount some kind of obstacle, possibly with checking existence of things on the generated page.

I know why I didn't want to make them cucumber specs: they're hard to write, hard to think about, hard to reuse steps for, and hard to check if a scenario isn't already tested somewhere. I think cucumber let us, or even encouraged us, to write a very unwieldy test suite, slow and full of redundancy. :/ I still have cucumber burnout, and every time I have to interact with them it takes me a month to be able to come back to the project. 🤷‍♀️

Reply:
I think we should wait for #3813 to be completed, then update this PR to add new tests to the "Many pseuds" Cucumber scenario, while removing `spec/requests/pseuds_spec.rb`:

- Generally we don't test pagination behavior (except for comments where we add a lot of custom behavior). We trust that `will_paginate` works, and instead focus on checking the order of list of things (which you've done in #3813).
- We resolved the other thread by using controller/model specs + cucumber tests, so let's not add a new type of spec here. Request specs shouldn't have access to Capybara matchers in the first place, that's something we add to every type of spec and I plan to remove (in #3936):

  https://github.com/otwcode/otwarchive/blob/bedbbef8c43585d97b5a56062355921e4c8f5fa7/spec/spec_helper.rb#L40